src/tests/stats/test_distribution_uniform.fypp

```diff
@@ -0,0 +1,282 @@
+#:include "common.fypp"
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
+#:set ALL_KINDS_TYPES = INT_KINDS_TYPES + RC_KINDS_TYPES
```

Comment:
Why not:
```suggestion
#:set ALL_KINDS_TYPES = INT_KINDS_TYPES + REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
```

Reply:
OK

Reply:
RC_KINDS_TYPES is needed for non-integer procedures.

Reply:
It could be. But `RC_KINDS_TYPES` is never used in test_distribution_uniform.fypp

Reply:
Yeh, you are right.